iothub_client/inc/iothub_client_options.h

```diff
@@ -41,6 +41,7 @@ extern "C"
     static STATIC_VAR_UNUSED const char* OPTION_MESSAGE_TIMEOUT = "messageTimeout";
     static STATIC_VAR_UNUSED const char* OPTION_BLOB_UPLOAD_TIMEOUT_SECS = "blob_upload_timeout_secs";
     static STATIC_VAR_UNUSED const char* OPTION_PRODUCT_INFO = "product_info";
+    static STATIC_VAR_UNUSED const char* OPTION_DT_MODEL_ID = "dt_model_id";
```

Comment:
>dt_model_id [](start = 63, length = 11)

Let's sync to talk about best name for this.  I'm concerned with underlying IoTHub layer "knowing" anything about DT concepts specifically (even though it's coded correctly RE no hard-dependency).